src/maintainer/knowledge_base.rst

```diff
@@ -1571,8 +1571,12 @@ The build section in the ``meta.yaml`` file should be something like this:
       - make test
       - make install VERBINST=1
 
-Notable here is ``INSTALLDIRS=vendor``, which selects the appropriate install location for a conda-forge package, and the ``NO_PERLLOCAL`` and ``NO_PACKLIST`` switches, which suppress various CPAN tracking files that are unnecessary when the module is being externally packaged via conda.
+Notable here is ``INSTALLDIRS=vendor``, which selects the appropriate install location for a conda-forge package, and the ``NO_PERLLOCAL`` and ``NO_PACKLIST`` switches, which suppress various CPAN tracking files (as described in the `ExtUtils::MakeMaker documentation <https://perldoc.perl.org/ExtUtils::MakeMaker>`__) that are unnecessary when the module is being externally packaged via conda.
 
 .. note::
 
   ``noarch: generic`` should be used only if the package is a pure Perl package.
+
+The other build system used for Perl packages is  ``Module::Build``. ``Module::Build`` is a system for building, testing, and installing Perl modules. 
+It is an alternative to ``ExtUtils::MakeMaker``. It does not require a ``make`` on your system - most of the ``Module::Build`` code is pure-perl and written in a very cross-platform way.
```

Comment:
```suggestion
It is an alternative to ``ExtUtils::MakeMaker``. It does not require ``make`` - most of the ``Module::Build`` code is pure Perl and written in a very cross-platform way.
```

I would add some examples here (maybe from `bioconda`). A quick search on the bioconda org shows [several useful results](https://cs.github.com/?scopeName=All+repos&scope=&q=org%3Abioconda+perl-module-build+path%3Ameta.yaml). For example:

* https://github.com/bioconda/bioconda-recipes/blob/master/recipes/perl-math-derivative/meta.yaml